mtags/src/main/scala/scala/meta/internal/pc/OverrideCompletions.scala

```diff
@@ -0,0 +1,506 @@
+package scala.meta.internal.pc
+
+import scala.meta.pc.PresentationCompilerConfig.OverrideDefFormat
+
+import org.eclipse.{lsp4j => l}
+
+import scala.collection.mutable
+import scala.collection.immutable.Nil
+
+trait OverrideCompletions { this: MetalsGlobal =>
+
+  private val DefaultIndent = 2
```

Comment:
defaultIndent (lowercase D)?

Reply:
I personally usually use upper camel case for constants https://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods, and [metals seem that it also follow this naming convention](https://github.com/scalameta/metals/blob/60e3d151dd80f0def1a122ff54e5c2f9e805e6cb/metals/src/main/scala/scala/meta/internal/metals/ClientExperimentalCapabilities.scala#L12).

I don't have a strong opinion around a naming convention, but what do you think about this? :)

Reply:
I don't mind either, let's leave it like this 👍